docs/quick_start.md

```diff
@@ -45,48 +45,133 @@ $ docker run -it --rm --gpus all \
+This will perform a search across various config parameters on the `add_sub` model. This takes over 40 minutes even on a TITAN RTX as Model Analyzer will try to find the search bounds automatically. When finished, Model analyzer stores all of the profiling measurements it has taken in a binary file in a directory called `./checkpoints` by default.
+
+```
+$ ls -l checkpoints
+total 12
+-rw-r--r-- 1 root root 11356 May 11 13:00 0.ckpt
+```
+
+Refer to the [Checkpoints](./checkpoints.md) documentation for more details on how checkpoint files work. 
+
+## Generate tables and summary reports using `analyze`
+In order to generate tables and summary reports, use the `analyze` subcommand as follows.
+
+```
+$ mkdir analysis results
+$ model-analyzer analyze --analysis-models add_sub -e analysis_results
+```
+
+This will fill the directory `./analysis_results` with measured data and a summary report. The directory should be structured as follows. 
 
 ```
 $HOME
   |--- model_analyzer
               |--- analysis_results
               .       |--- plots
-              .       |      |--- add_sub
-              .       |              |--- gpu_mem_v_latency.png
-              .       |              |--- throughput_v_latency.png
+              .       |      |--- simple
+              .       |      |      |--- add_sub
+                      |      |              |--- gpu_mem_v_latency.png
+                      |      |              |--- throughput_v_latency.png
+                      |      |
+                      |      |--- detailed
+                      |             |--- add_sub
+                      |                     |--- gpu_mem_v_latency.png
+                      |                     |--- throughput_v_latency.png
                       | 
                       |--- results
                       |       |--- metrics-model-inference.csv 
                       |       |--- metrics-model-gpu.csv 
                       |       |--- metrics-server-only.csv
                       |
                       |--- reports
-                              |--- add_sub
-                                      |--- result_summary.pdf
+                              |--- summaries 
+                              .        |--- add_sub
+                              .                |--- result_summary.pdf
+```
+
+## Generate a detailed report with `report`
+
+Model analyzer's `report` subcommand allows us to select one or more of the profiled configs for which to generate detailed reports containing a latency vs. throughput curve with a latency breakdown as well as other configurable plots. The detailed report also contains a table of measurements taken of that particular config. You can generate a detailed report for the two `add_sub` model configs `add_sub_i0` and `add_sub_i1` using:
```

Comment:
"Model analyzer's `report` subcommand allows you to examine the performance of a model config variant in detail. For example, it can show you the latency breakdown of your model to help you identify potential bottlenecks in your model performance."